content/cards/attendee/existing-projects.md

```diff
@@ -0,0 +1,9 @@
+### Does this project exist?
+
+Have you found solutions for the same challenge that exist already? Are there apps or solutions out there that are already being used to address this issue? If so, how do they fall short of meeting their goal, if at all? Be honest with yourself. Is it worth our time to build something that competes with what’s already out there? Are there ways we could improve on existing projects to be more impactful? If there’s nothing already out there that addresses this challenge, ask yourself why not. What barriers exist that inhibit progress in this area? Does this challenge call for a technical solution? Maybe the issue is at heart an organizational or physical one, rather than something that a tech approach can solve. 
+
+
+#### Identify existing alternatives 
+Take time to discuss as a group how people solve this problem today. Document all of the existing ways people solve the problem you want to solve.
+
+![leancanvas-box1](https://cloud.githubusercontent.com/assets/100216/12663467/b887dd18-c5e5-11e5-9ac9-06aa2736118f.png)
```

Comment:
Or, if we want the lean canvas step one here, we could maybe pull the other `#### Try a lean canvas model` into this. Maybe that is better.